### ISSUE TITLE
ApiGatewayProxyRequest.Resource not properly set using SAM Local

### ISSUE THREAD

**Issue by tfnaylor** [awslabs/aws-sam-cli#287]
Given a SAM template with the following configuration (partial):

```json
Resources:
  Create-Condition:
    Type: AWS::Serverless::Function
    Properties:
      Runtime: go1.x
      Handler: main
      CodeUri: .
      Events:
        Get-Condition:
          Type: Api
          Properties:
            Path: /api/conditions/{id}
            Method: GET
```
When invoking the endpoint locally using SAM Local and Postman as follows:

localhost:3000/api/conditions/12345

The value of the ApiGatewayProxyRequest.Resource property ends up as "/api/conditions/12345". When invoking the same operation against the template deployed to AWS, you get what seems like the correct value of "/api/conditions/{id}".

This seems like a bug to me and an important one if you are routing to handlers based on the resource path.

**Comment by kind3r** [comment]
Duplicate of #244

**Comment by jfuss** [comment]
This was corrected in the [0.3.0 release](https://github.com/awslabs/aws-sam-cli/releases/tag/v0.3.0). Closing since this has been addressed.
